docs/subcommands.md

```diff
@@ -0,0 +1,437 @@
+# Subcommands
+
+The `validator` command exposes the following subcommands.
+
+- [`describe`](#describe) - Describe the Validator results in a Kubernetes cluster.
+
+- [`install`](#install) - Install the Validator framework and configure Validator plugins.
+
+- [`uninstall`](#uninstall) - Uninstall the Validator framework and remove all Validator plugins.
+
+- [`upgrade`](#upgrade) - Upgrade Validator & reconfigure validator plugins.
+
+> [!WARNING]
+>
+> Credentials and other permissions may be required depending on the Validator plugins you use. For example, the AWS
+> plugin requires AWS credentials with elevated permissions to validate your AWS environment.
+
+## Install
+
+Use the `install` subcommand to install the Validator framework and configure Validator plugins. An interactive wizard
+will guide you through the installation process. You can also use a configuration file to install the Validator.
+
+> [!NOTE]
+>
+> A [kind](https://kind.sigs.k8s.io/) cluster will be deployed as part of the Validator installation. The name of the kind
+> cluster is `validator-kind-cluster`. You can install the Validator into an existing Kubernetes cluster by using the Helm chart. Refer to the
```

Comment:
`validatorctl` will also install validator into an existing cluster if you have the KUBECONFIG env var set. This is recommended.